lib/generators/rolemodel/github/templates/pull_request_template.md

```diff
@@ -15,6 +11,10 @@ What changed in this PR?
 * [ ] Change 2
 * [ ] ...
```

Comment:
Thoughts on removing these extras? I always need to delete them and add my own

Reply:
Hitting Enter after a todo adds a new one automatically, so I agree these are unnecessary.

Reply:
though, I guess this is the important one.  Isn't it?